.github/ISSUE_TEMPLATE/implementing-new-platform.md

```diff
@@ -0,0 +1,55 @@
+# Implementing a new supported platform
+
+## At Development
+Create PR's addressing the following:
+- [ ] [coreos-assembler](https://github.com/coreos/coreos-assembler)
+    - [Example PR](https://github.com/coreos/coreos-assembler/pull/2489)
+    - [ ] `Implement required functionality to support new platform`
```

Comment:
Hmm, is there a reason to make these items monospaced?

Reply:
NOOOPE, good question. It looks way better with it not being monospaced.

Reply:
Let me know if you like it spaced this way better!

Reply:
There's still a bunch of `monospaced` English text in the doc.  Can we remove it all?

Reply:
AH, I see it now. my editor is doing that.

Let me see if I can get that resolved.

Reply:
Done